packages/orchestration/src/exos/local-chain-account-kit.js

```diff
@@ -184,16 +185,17 @@ export const prepareLocalChainAccountKit = (
           };
           const { account: lca } = this.state;
           trace('lca', lca);
-          const delegatorAddress = await E(lca).getAddress();
+          const delegatorAddress = await V(lca).getAddress();
           trace('delegatorAddress', delegatorAddress);
-          const [response] = await E(lca).executeTx([
+          const [response] = await V(lca).executeTx([
             typedJson('/cosmos.staking.v1beta1.MsgUndelegate', {
               amount,
               validatorAddress,
               delegatorAddress,
             }),
           ]);
           trace('undelegate response', response);
+          // @ts-expect-error V is swallowing generic parameter?
           const { completionTime } = response;
```

Comment:
It seems we are losing the inferred return type with `V` here. If I hover over `executeTx` with `E`, I see:
```ts
(method) executeTx<(MsgUndelegate & {
    '@type': "/cosmos.staking.v1beta1.MsgUndelegate";
})[]>(messages: (MsgUndelegate & {
    '@type': "/cosmos.staking.v1beta1.MsgUndelegate";
})[]): PromiseVow<...>
```
With `V`:
```ts
(method) executeTx(messages: {
    '@type': string;
}[]): Promise<JsonSafe<{
    '@type': string;
}>[]>
```

Reply:
Created https://github.com/Agoric/agoric-sdk/issues/9497 so we can address this outside of this PR